core/bin/external_node/src/config/mod.rs

```diff
@@ -77,6 +79,22 @@ impl RemoteENConfig {
             .get_main_contract()
             .rpc_context("get_main_contract")
             .await?;
+        let base_token_addr = match client.get_base_token_l1_address().await {
+            Err(ClientError::Call(err))
+                if [
+                    ErrorCode::MethodNotFound.code(),
+                    // This what `Web3Error::NotImplemented` gets
+                    // `casted` into in the `api` server.
+                    ErrorCode::InternalError.code(),
+                ]
+                .contains(&(err.code())) =>
```

Comment:
Please be careful as on real envs we have proxy that may return different error code. Would be nice to check it, i.e. call some unexisting method on testnet and check that error code is in this array

Reply:
Discussed internally. We can merge + deploy a fix if this logic doesn't work